traitschema/schema.py

```diff
@@ -145,6 +145,14 @@ def to_hdf(self, filename, mode='w', compression=None, compression_opts=None):
                 trait = self.trait(name)
                 # tt = trait.trait_type
 
+                # Workaround for saving arrays containing unicode
+                if trait.array:
```

Comment:
I don't like this (see enthought/traits#390), but it appears like it works for the time being. Can you add a comment that summarizes this?